tests/library/video.spec.ts

```diff
@@ -475,7 +475,8 @@ it.describe('screencast', () => {
 
   it('should scale frames down to the requested size ', async ({ browser, browserName, server, headless, channel }, testInfo) => {
     it.fixme(!headless, 'Fails on headed');
-    it.fixme(browserName === 'chromium' && channel !== 'chromium-headless-shell', 'Fails on Chromiums');
+    const isHeadlessShell = channel === 'chromium-headless-shell' || (!channel && headless);
+    it.fixme(browserName === 'chromium' && !isHeadlessShell, 'Chromium (but not headless shell) has a min width issue');
```

Comment:
this condition is repeated so often throughout this, I wonder if it makes sense to add a fixture with its value in a follow-up PR.